bindings/python/docs/source/comparison.rst

```diff
@@ -0,0 +1,160 @@
+Quick Start
+===========
+
+This tutorial is intended as a comparison between using just PyMongo, versus
+with **PyMongoArrow**. The reader is assumed to be familiar with basic
+`PyMongo <https://pymongo.readthedocs.io/en/stable/tutorial.html>`_ and
+`MongoDB <https://docs.mongodb.com>`_ concepts.
+
+
+Reading Data
+^^^^^^^^^^^^
+The most basic way to read data using PyMongo is::
+
+.. code-block:: python
```

Comment:
`code-block:: python` isn't needed here because we already have `::` on the previous line.

Reply:
Fixed